### ISSUE TITLE
ES 1.X Support

### ISSUE THREAD

**Issue by cidthecoatrack** [logstash-plugins/logstash-input-elasticsearch#89]
This is a question, but it has been asked in the forums and went unanswered, and there is no concrete answer anywhere.  Does this plugin support reading inputs from an ES 1.X Index?  The earliest mentioned version is 2.2, and when I attempt to hit a 1.X index, I get the following error: `ElasticsearchIllegalArgumentException[Failed to decode scrollId]`.  If you explicitly don't support indexes that old, it is fine, but PLEASE specify it somewhere in the documentation.  If you do, please specify how to correct this issue.

**Comment by cidthecoatrack** [comment]
Update: looking through the release notes, I notice how 4.0.3 fixed the scrolling issue which broke scrolling in 1.X.  However, since you can't disable scrolling (as far as I can find), it effectively breaks all of 1.X compatibility.  Therefore, either there is a need to explicitly say 1.X is not supported because of scrolling, or to allow a user to disable scrolling in a request.

**Comment by andrewvc** [comment]
I agree, we should document this. Any support of Elasticsearch 1.x is purely accidental at this point.

**Comment by jordansissel** [comment]
@andrewvc agree we should document the minimum version of Elasticsearch that works (2.x?)

**Comment by jordansissel** [comment]
Elasticsearch 1.x is no longer developed and is unsupported. We don't test on it.

If you want Elasticsearch 1.x support, it might be best to fork this plugin and maintain 1.x support yourself.
